openchain-blockchain-coinprism/azuredeploy.json

```diff
@@ -0,0 +1,306 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
```

Comment:
Please see the latest guidance around preventing storageAccountName errors and ensuring they are unique https://github.com/Azure/azure-quickstart-templates/issues/944